src/wawa-table.ts

```diff
@@ -213,16 +213,16 @@ export class WawaTable extends LitElement {
     }
 
     protected render(): TemplateResult {
-        return html`${this.renderStyles()}<div style="width:100%;" @scroll=${this.onScroll}>
+        return html`${this.renderStyles()}<div class="table-container" style="width:100%;" @scroll=${this.onScroll}>
             <table part="table" style="border-collapse: collapse;width:100%;">
                 <thead part="head">
-                    ${this.headerTemplate}
+                    ${this.composeHeader(this.headerTemplate)}
                 </thead>
                 <tbody part="body">
                     ${this.startIndex > 0 ? html`<tr style="height:${this.startIndex * this.rowHeight}px"></tr>` : html``}
                     ${repeat(this.items, (i) => i.id, (i, index) => {
                         if (index >= this.startIndex && (this.visibleRows == 0 || index < this.startIndex + this.visibleRows)) {
-                            return html`${i.template}`;
+                            return html`${this.composeRow(index, i)}`;
```

Comment:
this allows a subclass to change how a row is rendered

Reply:
I don't think it's necessary to wrap the composeRow call inside html`` as composeRow already returns a TemplateResult